configJson.js

```diff
@@ -0,0 +1,24 @@
+const configBuilder = require('openfin-config-builder');
+
+if (process.argv.length <= 2) {
+    console.log('Usage: ' + __filename + ' <deployName>');
+    process.exit(-1);
```

Comment:
Typically `1` is used as the exit code for failure - any reason to use `-1` here?

https://nodejs.org/api/process.html#process_process_exit_code